server/src/routers/LocationsRouter/getOneLocation.js

```diff
@@ -1,21 +1,50 @@
 import getHours from "./getHours";
 
+import locations from "../../config/locations";
 import { E_BAD_LOC_REQ } from "../../config/constants";
 
-export default async function getOneLocation(data, query) {
+/**
+ *
+ * Gets one location's info, using cache for hours when possible
+ *
+ * @param  data [The data from the axios call to yale api]
+ * @param  query [The query with location info]
+ * @param  hoursDoc [The firebase doc object for locations/hours]
+ *
+ * @return {JS Object} [locations api info, such as hours, busyness, ...]
+ * 
+ */
+
+export default async function getOneLocation(data, query, hoursDoc) {
+    // Filter by data location
     const location = data.DATA.filter(
         entry => entry[data.COLUMNS.indexOf("ID_LOCATION")] == query.location
     );
     if (location.length) {
         const entry = location[0];
         let todayHours = undefined;
         let tomorrowHours = undefined;
+
+        // Use cache when possible
         try {
-            todayHours = await getHours(query.location, 0);
-            tomorrowHours = await getHours(query.location, 1);
+            todayHours= hoursDoc && 
```

Comment:
Small things: could you make sure there's a space after the assignment?

Reply:
Haha this was to align the && statements, but yes will add the space